### DISCUSSION TITLE
Use within Salesforce Program Management Module (PMM)

### DISCUSSION THREAD

**Discussion by CAMERONBIACK** [deejay-hub/timeline-lwc#306] [Q&A] [answered]
Hi All,

I have installed Time Warp and added onto our contact record lightning page. We use PMM within our organisation to record a persons interactions with our charity. 

Each time someone attends, a record is created within the PMM Service Delivery object (pmdm__ServiceDelivery__c). I would like to show each service delivery on the Time Warp timeline. 

_**Example with dummy data**_
![image](https://github.com/user-attachments/assets/0973eedb-7416-438c-87e8-28174429912c)

The service delivery records already pull through onto the contact record in a related list. However, I think where I am having issues is that on the service delivery record the contact is listed as 'Client' via a lookup field rather than 'Contact'. 

![image](https://github.com/user-attachments/assets/819966cb-7d8a-48ea-ae68-1589fd5dc3c9)

I think I have gotten most of the way there but it's the relationship that I am struggling with. I consulted the similar discussions and #166 got me most of the way to this point.

![image](https://github.com/user-attachments/assets/b3aafc25-0ebd-4891-b81f-9e0fa40ce00a)

Any support would be amazing as there are other objects I'd love to also bring into the timeline to show the full breath of support someone is getting from our charity. I feel like once I get this one in my head - I'll be off and running.

Appreciate your support and guidance!

**Comment by CAMERONBIACK** [comment, marked as answer]
I think I have managed to solve it - I ignored your advice on the previous #166 and I should have updated the relationship to pmdm__Service_Deliveries__r

**Reply by deejay-hub** [reply]
These are the best discussions to have! Glad you found the solution
